app/src/main/java/com/teamltt/carcare/activity/HomeActivity.java

```diff
@@ -146,27 +147,6 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return true;
     }
 
-    protected void goToDemo(View view) {
-        Intent intent = new Intent(this, DemoActivity.class);
-        startActivity(intent);
-    }
-    protected void goToStatic(View view) {
-        Intent intent = new Intent(this, DemoActivity.class);
-        startActivity(intent);
-    }
-    protected void goToDynamic(View view) {
-        Intent intent = new Intent(this, DemoActivity.class);
-        startActivity(intent);
-    }
-
-    /*protected void openDrawer(View view) {
-        if (drawer.isDrawerOpen(findViewById(android.R.id.home))) {
-            drawer.closeDrawer(Gravity.LEFT);
-        }
-        else {
-            drawer.openDrawer(Gravity.RIGHT);
-        }
-    }*/
     public boolean onOptionsItemSelected(MenuItem item) {
```

Comment:
As of now, there could be a conflict in how this method handles an item selection and how the super method does. If both handle the same `item.getItemId()` then it may be handled twice and duplicate work is being done.